asm/non_matchings/code/code_801031F0/func_801032B0.s

```diff
@@ -1,69 +1,12 @@
-.include "macro.inc"
+.rdata
```

Comment:
.rdata is specifically for strings, while .late_rodata is for the rest of rodata like floats/doubles.
```suggestion
.late_rodata
```